core/src/main/java/org/hibernate/ogm/jpa/impl/OgmEntityManager.java

```diff
@@ -633,39 +616,4 @@ public CriteriaBuilder getCriteriaBuilder() {
 	public Metamodel getMetamodel() {
 		return hibernateEm.getMetamodel();
 	}
-
-	/**
```

Comment:
Hum, `ExceptionMapper` was accidentally gone in ORM at some point but I think it was re-added later on. I'll see whether I can re-establish this. Just seeing this now.